.github/workflows/stale.yml

```diff
@@ -0,0 +1,31 @@
+name: Stale PRs
+on:
+  schedule:
+    - cron: '0 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Label/close stale PRs
+        uses: actions/stale@v8
+        with:
+          stale-pr-message: 'This PR is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 10 days.'
```

Comment:
Do I understand correctly that when removing the stale label there will be another 60 days before this message will be added again?

Reply:
True, any kind of activity, including removing the label or simply commenting on the PR will reset the timer to 0.